File: projects/number-guessing/ruby/guess.rb
secret = rand(1..100)
attempts = 0

puts "Guess the number (1-100)"

loop do
  print "Your guess: "
  guess = gets.chomp.to_i
  attempts += 1

  if guess < secret
    puts "Too low"
  elsif guess > secret
    puts "Too high"
  else
    puts "Correct! Attempts: #{attempts}"
    break
  end
end

File: projects/number-guessing/csharp/Guess.cs
using System;

class Guess {
    static void Main() {
        Random rand = new Random();
        int secret = rand.Next(1, 101);
        int attempts = 0;

        Console.WriteLine("Guess the number (1-100)");

        while (true) {
            Console.Write("Your guess: ");
            int guess = int.Parse(Console.ReadLine());
            attempts++;

            if (guess < secret) Console.WriteLine("Too low");
            else if (guess > secret) Console.WriteLine("Too high");
            else {
                Console.WriteLine($"Correct! Attempts: {attempts}");
                break;
            }
        }
    }
}
